lib/usbd/backend/usbd_stm32_otg_fs.c

```diff
@@ -78,16 +78,20 @@ static usbd_device *init(const usbd_backend_config *config)
 	_usbd_dev.backend = &usbd_stm32_otg_fs;
 	_usbd_dev.config = config;
 
+	//* stm32f4 use FS CID=0x1100, HS CID=0x02000600
+	//* stm32f7            0x3000,    CID=0x00003100
+	const unsigned otg_hs_cid_boundary = 0x3000;
+
 	if (config->feature & USBD_VBUS_SENSE) {
-		if (OTG_FS_CID >= 0x00002000) { /* 2.0 HS core*/
+		if (OTG_FS_CID >= otg_hs_cid_boundary) { /* 2.0 HS core*/
```

Comment:
`OTG_FS_CID >= 0x00003000` leads to running 1.0 HS core init code on 2.0 HS core init